### PR TITLE
fix #25 (externalize js)

### LINKED ISSUE #25: Insufficient compatibility with HTML 5 and Plone 5.2: orderedSelectionList.pt uses inline JS with special characters
In a Plone 5.2 context, the use of `orderedSelectionList.pt` causes a JS syntax error.

The template uses inline JS and `>` in its function `moveItems`. Plone 5.2's "output transform chain" uses XML technologies which transform the `>` into `&gt;`. Plone adds a `DOCTYPE XHTML` declaration and the corresponding DTD declares the `scrpt` content as `#PCDATA`; thus, the `&gt;` should be equicalent to `>`. Apparently, however, (at least some) modern browsers follow the HTML5 spec which states: a page with content type `text/hmll` is parsed according to the HTML5 specification - and this declares the script content as `#CDATA`. In those browsers `&gt;` is not equivalent to `>` and can lead to a JS syntax error.

`orderedSelectionList.pt` should follow the XHTML recommendation and avoid inline JS with special characters.

### PR DESCRIPTION
**Pull request by d-maurer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def alltests():
           'pytz',
           'zope.browser>=1.1',
           'zope.browserpage>=3.11.0',
+          'zope.browserresource',
```
Comment (mgedmin) [inline setup.py]: So now I've got to ask: don't we need a ZCML directive to register the zope.formlib resource directory?

I'd assumed we had all that done before and you've just added a couple of files into the already-registered directory, but this change raised doubts.

Comment (mgedmin) [reply setup.py]: Never mind, somehow I've overlooked the change to configure.zcml!

**Comment by d-maurer** [comment]
@icemac How can the test coverage fall due to a change in untested page templates only? I am inclined to ignore the coverage decrease by 0.0004 %. Do you agree?

**Comment by d-maurer** [comment]
> (I assume you've tested that the widgets work manually, since we have no automated tests for Javascript.)

In fact, I tested (manually) only the proper working of the `orderedSelectionList.pt` which was the reason for #25 (and for which I have an application which uses it). I modified `pageform.pt` and `subpageform.pt` analogously (externalize inline JS) and checked that the `++resource++` used there  works but have not further tested them.

**Comment by mgedmin** [comment]
I do not trust coveralls.io and tend to ignore its warnings.

**Comment by d-maurer** [comment]
@icemac Can we have a new release -- despite a (unexplained) drop of test coverage by 0.0004 %?

**Comment by jugmac00** [comment]
@d-maurer and @icemac - please have a look at https://github.com/zopefoundation/zope.formlib/issues/28 before releasing a new package

**Comment by mgedmin** [comment]
Also, it's about time to drop Python 3.4 compat and maybe add 3.8.

**Comment by jugmac00** [comment]
@mgedmin This was exactly my plan... thats why I run tox and noticed it is broken.

**Comment by mgedmin** [comment]
Looks like I beat you to it: https://github.com/zopefoundation/zope.formlib/pull/29 ;)

**Comment by icemac** [comment]
I am going to create a release now.

**Comment by icemac** [comment]
I just released https://pypi.org/project/zope.formlib/4.7.0/.

**Comment by d-maurer** [comment]
Michael Howitz wrote at 2020-2-27 01:03 -0800:
>I just released https://pypi.org/project/zope.formlib/4.7.0/.

Thank you!

**Pull request merged**